dtfs-central-api/src/types/utilisation-reports.d.ts

```diff
@@ -8,12 +8,36 @@ export type UtilisationReportReconciliationStatus =
 
 export type UtilisationReportReconciliationSummaryItem = {
```

Comment:
I think this type may have had most of its properties removed in a merge? It should be of type:
```
export type UtilisationReportReconciliationSummaryItem = {
  reportId?: ObjectId;
  bank: {
    id: string;
    name: string;
  };
  status: UtilisationReportReconciliationStatus;
  dateUploaded?: Date;
  totalFacilitiesReported?: number;
  facilitiesLeftToReconcile?: number;
};
```

Reply:
Yeh I'm not sure how that happened - oh well!